doc/ref_model/chapters/chapter04.md

```diff
@@ -478,7 +478,7 @@ n100, n200, n300, n400, n500, n600 | N | Y | N
 | `e.cap.016`<br />(FPGA/other Acceleration H/W)                        | Yes (if offered)         | No                       | || | `e.cap.017`<br />(Monitoring of L2-7 data)                  | No                       | Yes                      | Exposed monitoring capabilities as per [**Table 4-3**](#Table4-3)   |
 | `i.cap.014`<br />(CPU cores consumed by the NFVI on the worker nodes) | any                      | any                      | |
 | `i.cap.015`<br />(Memory consumed by NFVI on the worker nodes)        | any                      | any                      | |
-| `i.cap.016`<br />(CPU allocation ratio)                     | 4:1                      | 1:1                      | Internal SLA capabilities as per [**Table 4-6**.](#Table4-6)        |
+| `i.cap.016`<br />(CPU allocation ratio)                     | 1:1                      | 1:1                      | Internal SLA capabilities as per [**Table 4-6**.](#Table4-6)        |
```

Comment:
Suggest adding a note below the table stating that this is to allow for future performance benchmarking, allowing operators to modify for actual deployments if they are willing to accept the risk of an impact on performance.